b2share/config.py

```diff
@@ -98,6 +266,17 @@
 )
 
 
+#: Endpoint for uploading files.
+DEPOSIT_FILES_API = u'/api/files'
+#: Template for deposit list view.
+DEPOSIT_SEARCH_API = '/api/deposit/depositions'
+#: Template for deposit records API.
+DEPOSIT_RECORDS_API = '/api/deposit/depositions/{pid_value}'
+#: Records REST API endpoints.
+RECORDS_API = '/api/records/{pid_value}'
+#: Default API endpoint for search UI.
+SEARCH_UI_SEARCH_API = "/api/records/"
+
```

Comment:
HK: These might not be required. 

HH: Might be related to first commits conserning flask_app building.